proposals/cloud_detection.json

```diff
@@ -0,0 +1,65 @@
+{
+    "id": "cloud_detection",
+    "summary": "Cloud detection",
+    "description": "Detects clouds and cloud shadow in optical imagery. By default, the cloud mask is applied to the input data cube. Set `apply_mask` to `false` to add the cloud mask as a new band.",
```

Comment:
This description previously mentioned that a band is (always) added, but that conflicted with the apply_mask parameter. I changed it to comply with the parameter. If the desired behavior should be different, please change the description and parameter accordingly.